golog.go

```diff
@@ -206,25 +213,34 @@ type Logger interface {
 	AsStdLogger() *log.Logger
 }
 
+// shouldEnableTrace returns true if tracing was enforced through a linker
+// flag, or if TRACE=true is set in the environment, while favoring the former.
+func shouldEnableTrace() bool {
+	if strings.ToLower(enableTraceThroughLinker) == "true" ||
+		enableTraceThroughLinker == "1" {
+		return true
+	}
+
+	var v bool
+	v, _ = strconv.ParseBool(os.Getenv("TRACE"))
+	if v == true {
+		return true
+	}
+
+	return false
+}
+
 func LoggerFor(prefix string) Logger {
 	l := &logger{
 		prefix: prefix + ": ",
 	}
 
-	trace := os.Getenv("TRACE")
-	l.traceOn, _ = strconv.ParseBool(trace)
-	if !l.traceOn {
-		prefixes := strings.Split(trace, ",")
```

Comment:
This logic allowed enabling tracing just for specific loggers. It's a shame to lose that.

Reply:
I've included that functionality and documented its usage: https://github.com/getlantern/golog/commit/c622d0d5def893e23b62853d53bcd13ce66c5460